linera-views/src/test_utils/mod.rs

```diff
@@ -773,3 +773,71 @@ where
     }
     assert_eq!(keys, read_keys);
 }
+
+/// A store can be in exclusive access where it stores the absence of values
+/// or in shared access where only values are stored and (key, value) once
+/// written are never modified nor erased.
+///
+/// In case of no exclusive access the following scenarion is checked
+/// * Store 1 deletes a key and does not mark it as missing in its cache.
+/// * Store 2 writes the key
+/// * Store 1 reads the key, but since it is no in the cache it can read
+///   it correctly.
+pub async fn no_exclusive_access_admin_test<S: TestKeyValueStore>()
+where
+    S::Error: Debug,
+{
+    let config = S::new_test_config().await.expect("config");
+    let namespace = generate_test_namespace();
+    S::create(&config, &namespace).await.expect("creation");
+    let key = vec![42];
+    //
```

Comment:
nit: We use simple empty lines in the rest of the code.

Reply:
Ok, done.